packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -424,21 +438,9 @@ export function TransferPanelMain({
   const [, setQueryParams] = useArbQueryParams()
 
   useEffect(() => {
-    const l2ChainId = isConnectedToArbitrum ? externalFrom.id : externalTo.id
-
     setFrom(externalFrom)
     setTo(externalTo)
-
-    // Keep the connected L2 chain id in search params, so it takes preference in any L1 => L2 actions
-    setQueryParams({ l2ChainId })
```

Comment:
We shouldn't need it anymore. Please test the network switching to make sure all is correct.